### DISCUSSION TITLE
Fetching the pending header is not supported - eth_call(pending) unsupported

### DISCUSSION THREAD

**Discussion by dankostiuk** [0xPolygon/polygon-edge#522] [Ideas]
### Description

JSON-RPC calls that include the "pending" block tag such as eth_call are not supported. For example:
```
{"jsonrpc": "2.0", "method": "eth_call", "params": [{"to": "0x2d8CD9969597A67829aEb63e953fB791401dc4C2", "data": "0xa0e67e2b"}, "pending"], "id": 180}
```

gives the response:
```
{
    "jsonrpc": "2.0",
    "id": 1,
    "error": {
        "code": -32600,
        "message": "fetching the pending header is not supported"
    }
}
```

As the pending block tag is used by many dApps, it would be greatly appreciated if this call could be supported.

### Environment
- Ubuntu 20
- Polygon-Edge 0.1.0

**Comment by laviniat1996** [comment]
Hi @dankostiuk ,

This answer comes a bit late, I apologize for this. At the moment, it is true that **Polygon Edge** does not support the **pending** block tag, and we don't know yet if we'll add it in the future.
If we'll do, I will give you an update here.
